package.json

```diff
@@ -66,17 +66,21 @@
     "react-colorful": "^4.4.4",
     "react-confetti": "^6.0.0",
     "react-dom": "^16.12.0",
+    "react-device-detect":"^1.17.0",
     "react-helmet": "^5.2.1",
     "react-icons": "^3.9.0",
     "react-live": "^2.2.2",
+    "react-loadable":"^5.5.0",
     "react-loader-spinner": "^3.1.14",
     "react-responsive": "^8.0.3",
     "react-spring": "^8.0.27",
     "react-three-fiber": "^5.3.18",
     "react-tooltip": "^4.2.5",
     "react-transition-group": "^4.4.1",
     "react-typography": "^0.16.19",
+    "reactour":"^1.18.3",
     "react-use": "^15.3.8",
+    "styled-components":"^4.0.0",
```

Comment:
@harsh242 This lib is not being used anywhere. Also, remove it.

Reply:
[Reactour](https://github.com/elrumordelaluz/reactour) requires styled components

Reply:
Got it.